REQUIRE

```diff
@@ -1,6 +1,8 @@
 julia 1.0
 Optim 0.16.0
 PDMats 0.9.4
+ElasticPDMats 0.1.0
```

Comment:
Could we make these imports optional? Many users won't need this functionality. I don't actually know how this is done in julia, but it seems that [Requires.jl](https://github.com/MikeInnes/Requires.jl) is one way to do it.

Reply:
This should be rather simple to do. The GPE constructors would need to be changed and then one could load GPEelastic.jl using Requires. `ElasticPDMats` is rather small, however.
Would you still load `ElasticPDMats` for testing?

Reply:
OK, fair point, I was overthinking. Let's have `ElasticPDMats` as a dependency.